bridge_ui/src/utils/metaMaskChainParameters.ts

```diff
@@ -0,0 +1,148 @@
+// https://docs.metamask.io/guide/rpc-api.html#wallet-addethereumchain as per EIP-3085
+export interface AddEthereumChainParameter {
+  chainId: string; // A 0x-prefixed hexadecimal string
+  chainName: string;
+  nativeCurrency: {
+    name: string;
+    symbol: string; // 2-6 characters long
+    decimals: 18;
+  };
+  rpcUrls: string[];
+  blockExplorerUrls?: string[];
+  iconUrls?: string[]; // Currently ignored.
+}
+
+// https://chainid.network/chains.json for chain info
+export const METAMASK_CHAIN_PARAMETERS: {
+  [evmChainId: number]: AddEthereumChainParameter;
+} = {
+  1: {
+    chainId: "0x1",
+    chainName: "Ethereum Mainnet",
+    nativeCurrency: { name: "Ether", symbol: "ETH", decimals: 18 },
+    rpcUrls: ["https://rpc.ankr.com/eth"],
+    blockExplorerUrls: ["https://etherscan.io"],
+  },
+  3: {
+    chainId: "0x3",
+    chainName: "Ropsten",
+    nativeCurrency: { name: "Ropsten Ether", symbol: "ROP", decimals: 18 },
+    rpcUrls: ["https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161"],
+    blockExplorerUrls: ["https://ropsten.etherscan.io"],
+  },
+  5: {
+    chainId: "0x5",
+    chainName: "Görli",
+    nativeCurrency: { name: "Görli Ether", symbol: "GOR", decimals: 18 },
+    rpcUrls: ["https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161"],
+    blockExplorerUrls: ["https://goerli.etherscan.io"],
+  },
+  56: {
+    chainId: "0x38",
+    chainName: "Binance Smart Chain Mainnet",
+    nativeCurrency: {
+      name: "Binance Chain Native Token",
+      symbol: "BNB",
+      decimals: 18,
+    },
+    rpcUrls: ["https://bsc-dataseed.binance.org"],
+    blockExplorerUrls: ["https://bscscan.com"],
+  },
+  97: {
+    chainId: "0x61",
+    chainName: "Binance Smart Chain Testnet",
+    nativeCurrency: {
+      name: "Binance Chain Native Token",
+      symbol: "BNB",
+      decimals: 18,
+    },
+    rpcUrls: ["https://data-seed-prebsc-1-s1.binance.org:8545"],
+    blockExplorerUrls: ["https://testnet.bscscan.com"],
+  },
+  137: {
+    chainId: "0x89",
+    chainName: "Polygon Mainnet",
+    nativeCurrency: { name: "MATIC", symbol: "MATIC", decimals: 18 },
+    rpcUrls: ["https://polygon-rpc.com"],
+    blockExplorerUrls: ["https://polygonscan.com"],
+  },
+  250: {
+    chainId: "0xfa",
+    chainName: "Fantom Opera",
+    nativeCurrency: { name: "Fantom", symbol: "FTM", decimals: 18 },
+    rpcUrls: ["https://rpc.ftm.tools"],
+    blockExplorerUrls: ["https://ftmscan.com"],
+  },
+  686: {
+    chainId: "0x2ae",
+    chainName: "Karura Network",
+    nativeCurrency: { name: "Karura Token", symbol: "KAR", decimals: 18 },
+    // TODO: add rpcUrl
+    rpcUrls: [],
```

Comment:
Couldn't find any valid Karura and Acalla RPC urls, so these will fail to add. Even chainlist.org doesn't support adding them.

Reply:
Updating :)